x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/filtering/FilteringValidation.java

```diff
@@ -69,6 +69,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.stringListField(IDS_FIELD.getPreferredName(), ids);
             builder.stringListField(MESSAGES_FIELD.getPreferredName(), messages);
         }
+        builder.endObject();
```

Comment:
Did we've a bug here? If yes, does this need to be backported?

Reply:
yes we did, yeah I'm looking into creating a backport PR !

Reply:
https://github.com/elastic/elasticsearch/pull/107467